open-company/open-company-api
# Section manipulation

https://trello.com/c/y3utBhod

This PR adds support for PATCHING the :sections property of a company to support reordering and removing sections.

To test:

0. Review the codes
1. Review the `./integration/section/section_manipulation.clj` tests
2. Execute all the tests with `lein test!` (or look at the CI results)
3. Run the cURL command from the README that has you reorder Buffer's sections. Then GET buffer (or use the UI) and confirm that the sections are reordered.
3. Run the cURL command from the README that has you remove a section (marketing) from Buffer. Then GET buffer (or use the UI) and confirm that the section is removed.
5. Merge
6. Go build a reorder section and remove section UI in the ToC
7. Learn Japanese
